src/app/components/addresses/Residents/Resident.tsx

```diff
@@ -1,5 +1,5 @@
 import useValues from "./hooks/useValues"
-import PersonDisplay from "app/components/shared/PersonDisplay/PersonDisplay"
+import { PersonNameDisplay } from "@amsterdam/wonen-ui"
```

Comment:
Line 2 and 3 could be combined: `import { PersonNameDisplay, DefinitionList } from "@amsterdam/wonen-ui"`

Reply:
Fixed it